react/components/FacetCheckboxListItem.js

```diff
@@ -1,5 +1,5 @@
 import classNames from 'classnames'
-import React from 'react'
+import React, { useMemo } from 'react'
```

Comment:
Is this new import being used?

Reply:
Yes, this is. I don't know how this file worked before, because the useMemo was in use(line 29) but the import didn't exist. It only broke when I used the filter in mobile mode.